tests/components/deconz/test_light.py

```diff
@@ -1486,11 +1511,14 @@ async def test_verify_group_color_mode_fallback(
                 "type": "Dimmable light",
                 "uniqueid": "00:17:88:01:08:11:22:33-01",
             },
-        },
-    }
-    with patch.dict(DECONZ_WEB_REQUEST, data):
-        await setup_deconz_integration(hass, aioclient_mock)
-
+        }
+    ],
+)
+@pytest.mark.usefixtures("config_entry_setup")
+async def test_verify_group_color_mode_fallback(
+    hass: HomeAssistant, mock_deconz_websocket
```

Comment:
https://github.com/home-assistant/core/pull/120967#discussion_r1665452557

Reply:
Will do in separate PR